src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -111,18 +109,17 @@ private static OnnxTransform Create(IHostEnvironment env, ModelLoadContext ctx)
         private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, ISchema inputSchema)
             => Create(env, ctx).MakeRowMapper(Schema.Create(inputSchema));
 
-        private OnnxTransform(IHostEnvironment env, Arguments args, byte[] modelBytes = null)
+        private OnnxTransform(IHostEnvironment env, Arguments args, byte[] modelBytes = null) :
+            base(Contracts.CheckRef(env, nameof(env)).Register(nameof(OnnxTransform)))
         {
-            Contracts.CheckValue(env, nameof(env));
-            _host = env.Register(RegistrationName);
-            _host.CheckValue(args, nameof(args));
-            _host.CheckNonWhiteSpace(args.InputColumn, nameof(args.InputColumn));
-            _host.CheckNonWhiteSpace(args.OutputColumn, nameof(args.OutputColumn));
+            Host.CheckValue(args, nameof(args));
+            Host.CheckNonWhiteSpace(args.InputColumn, nameof(args.InputColumn));
```

Comment:
>CheckNonWhiteSpace [](start = 17, length = 18)

is this the best check? Would CheckNonNull be better?

Reply:
I doubt you can have layer in Onnx which named as " ", also it was code before my change so i hope it's valid :)